test/LibraryManager.Test/ManifestTest.cs

```diff
@@ -235,7 +237,7 @@ public void Cleanup()
             IEnumerable<ILibraryOperationResult> result = await manifest.RestoreAsync(CancellationToken.None);
 
             Assert.AreEqual(1, result.Count());
-            Assert.IsTrue(result.Last().Errors.Any(e => e.Code == "LIB016"), "LIB016 error code expected.");
+            Assert.IsTrue(result.Last().Errors.Any(e => e.Code == "LIB019"), "LIB019 error code expected.");
```

Comment:
>LIB019 [](start = 67, length = 6)

This test needs to be updated to continue validating a LIB016, and a new test should be covering LIB019